eng/testing/linker/project.csproj.template

```diff
@@ -75,6 +75,18 @@
     {AdditionalProjectReferences}
   </ItemGroup>
 
+  <Target Name="RemoveInvariantGlobalization" BeforeTargets="_SetWasmBuildNativeDefaults" Condition="'$(TargetArchitecture)' == 'wasm'">
+    <ItemGroup>
+      <_BoolPropertiesThatTriggerRelinking Remove="InvariantGlobalization" />
+    </ItemGroup>
+  </Target>
+  <PropertyGroup Condition="'$(TargetArchitecture)' == 'wasm'">
```

Comment:
Do we need to register the target? The execution should be enforced by `BeforeTargets="_SetWasmBuildNativeDefaults"`